<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::LauncherWindow</class>
 <widget class="QDialog" name="GammaRay::LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gammaray.qrc">
    <normaloff>:/gammaray/GammaRay-128x128.png</normaloff>:/gammaray/GammaRay-128x128.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="GammaRay::AttachDialog" name="attachPage">
      <attribute name="title">
       <string>Attach</string>
      </attribute>
     </widget>
     <widget class="GammaRay::LaunchPage" name="launchPage">
      <attribute name="title">
       <string>Launch</string>
      </attribute>
     </widget>
     <widget class="GammaRay::SelfTestPage" name="selfTestPage">
      <attribute name="title">
       <string>Self Test</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources/gammaray.qrc">:/gammaray/GammaRay-128x128.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="aboutLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;GammaRay %1&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The Qt application inspection and manipulation tool. Learn more at &lt;a href=&quot;http://www.kdab.com/gammaray&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;http://www.kdab.com/gammaray&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;Copyright (C) 2010-2012 Klar√§lvdalens Datakonsult AB, a KDAB Group company, &lt;a href=&quot;mailto:info@kdab.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;info@kdab.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Authors:&lt;/span&gt;&lt;br/&gt;Allen Winter &amp;lt;allen.winter@kdab.com&amp;gt;&lt;br/&gt;Andreas Holzammer &amp;lt;andreas.holzammer@kdab.com&amp;gt;&lt;br/&gt;David Faure &amp;lt;david.faure@kdab.com&amp;gt;&lt;br/&gt;Kevin Funk &amp;lt;kevin.funk@kdab.com&amp;gt;&lt;br/&gt;Milian Wolff &amp;lt;milian.wolff@kdab.com&amp;gt;&lt;br/&gt;Patrick Spendrin &amp;lt;patrick.spendrin@kdab.com&amp;gt;&lt;br/&gt;Stephen Kelly &amp;lt;stephen.kelly@kdab.com&amp;gt;&lt;br/&gt;Till Adam &amp;lt;till@kdab.com&amp;gt;&lt;br/&gt;Thomas McGuire &amp;lt;thomas.mcguire@kdab.com&amp;gt;&lt;br/&gt;Tobias Koenig &amp;lt;tobias.koenig@kdab.com&amp;gt;&lt;br/&gt;Volker Krause &amp;lt;volker.krause@kdab.com&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="GammaRay::PromoLabel" name="promoLabel" native="true"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::PromoLabel</class>
   <extends>QWidget</extends>
   <header>promolabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::AttachDialog</class>
   <extends>QWidget</extends>
   <header>attachdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::LaunchPage</class>
   <extends>QWidget</extends>
   <header>launchpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::SelfTestPage</class>
   <extends>QWidget</extends>
   <header>selftestpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/gammaray.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GammaRay::LauncherWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GammaRay::LauncherWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
